src/shared/components/comment/comment-nodes.tsx

```diff
@@ -27,37 +26,45 @@ interface CommentNodesProps {
   hideImages?: boolean;
 }
 
-export class CommentNodes extends Component<CommentNodesProps, any> {
-  constructor(props: any, context: any) {
-    super(props, context);
-  }
-
-  render() {
-    let maxComments = this.props.maxCommentsShown ?? this.props.nodes.length;
-
-    return (
-      <div className="comments">
-        {this.props.nodes.slice(0, maxComments).map(node => (
-          <CommentNode
-            key={node.comment_view.comment.id}
-            node={node}
-            noBorder={this.props.noBorder}
-            noIndent={this.props.noIndent}
-            viewOnly={this.props.viewOnly}
-            locked={this.props.locked}
-            moderators={this.props.moderators}
-            admins={this.props.admins}
-            markable={this.props.markable}
-            showContext={this.props.showContext}
-            showCommunity={this.props.showCommunity}
-            enableDownvotes={this.props.enableDownvotes}
-            viewType={this.props.viewType}
-            allLanguages={this.props.allLanguages}
-            siteLanguages={this.props.siteLanguages}
-            hideImages={this.props.hideImages}
-          />
-        ))}
-      </div>
-    );
-  }
-}
+export const CommentNodes = ({
+  allLanguages,
+  nodes,
+  siteLanguages,
+  viewType,
+  admins,
+  enableDownvotes,
+  hideImages,
+  locked,
+  markable,
+  maxCommentsShown = nodes.length,
```

Comment:
Why change this?

Reply:
I can't remember why entirely TBH. I'll revert this change.